module/VuFind/src/VuFind/Config/Locator.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * VF Configuration Locator
+ * VuFind Configuration Locator - A static compatibility wrapper around PathResolver
```

Comment:
It's probably worth flagging all the methods in this class deprecated.

Reply:
Done.